www/grpc/swagger-ui/pactus.swagger.json

```diff
@@ -242,7 +264,7 @@
         ]
       }
     },
-    "/v1/blockchain/validator/{number}": {
+    "/v1/blockchain/validator/number/{number}": {
```

Comment:
Please change router path base on restful API relation standard:

```
/v1/blockchain/validator/numbers/{number}
```

Reply:
Why it should be plural? Because it queries one block only

Reply:
Are mean get validator by number?

Reply:
Yes, sorry. I mean one validator.

Reply:
> Yes, sorry. I mean one validator.
> 

Better design:

```
v1/blockchain/validators/{id,hash,number and etc}
```

Reply:
@Ja7ad 
I have two questions:

1- This function returns only one validator. Why should we use the plural "validators"?
2- Don't you think defining different routes is better than merging all in one route? This approach aligns with the Proto APIs: we have one API to fetch a validator by number and another to fetch by address.

Reply:
1. this is standard of restful design 1 in many validators.
2. you can separate for SRP, but don't required, because you can check input with regex for number or address in business.

for separate:
- `v1/blockchain/validators/number/{number}`
- `v1/blockchain/validators/address/{address}`

or use query params for single router:

- `v1/blockchain/validator?number=xx`
- `v1/blockchain/validator?address=xx`